index.html

```diff
@@ -87,58 +87,75 @@ <h2 class="section__heading__text">
         class="about__img" />
     </section>
 
-    <!-- <section class="about">
-      <div class="about__container">
-        <h2 class="about__container__heading">
-          <span class="about__container__heading__content--blue">01.</span>
-          About
-        </h2>
-        <div class="about_container__description">
-          <p>Hello, I'm Iwona and I'm an entry-level frontend developer.</p>
-          <br />
-          <p>
-            My journey to become a developer started in 2022. In 2023 I finished
-            postgraduate studies - Frontend web development. Before that I was
-            learning by myself, using quality courses and lectures.
-          </p>
-          <br />
-          <p>
-            Also in 2023 I had the opportunity to participate in Dare IT
-            program, which is ment for woman who want to join IT. Dare IT
-            Mentoring is a 4-month program where I work with my mentor on my
-            Frontend Developer skills. In addition to 1:1 sessions with a
-            mentor, the Program offers: HR consultations, thematic webinars,
-            networking meetings, all in a community of almost 300 people:
-            mentors, HR consultants, Program Partners and other mentees. Besides
-            that I also work on my portfolio and together with real team we
-            create a website (coming soon). Nearly 1,000 candidates applied for
-            the 120 spots - I'm very happy and thankful for this opportinity.
-          </p> <br>
-          <p>
-            My plans for 2024 are pretty awsome, let's keep in touch!
-          </p>
-        </div>
-        <img src="https://cdn.pixabay.com/photo/2017/08/05/13/06/girl-2583442_1280.jpg" alt="photo of a girl" class="about__img">
+  <section class="experience section">
+      <div class="section__heading">
+        <h2 class="section__heading__text">
+          <span class="section__container__text--blue">02.</span>
+          Experience</h2>
       </div>
-    </section> -->
 
-    <section class="experience">
+      <div class="experience__container"> 
+        <div class="experience__mini-box">
+      </div>
+      <h5 class="experience__content">DARE IT Mentoring Program - Frontend Developer</h5>
+      <p class="experience__date">December 2023 - March 2024</p>
+      <p class="experience__description">- Write performant and maintainable code. <br />
+      - Communicate and work with multi-disciplinary team of UI designer, UX designer, PM & tester.<br />
+      - Work with team on a real project - coming soon. <br />
+    - Create this portfolio based on project from Figma. <br />
+   </p>
+      </div>
+      <div class="line"></div>
```

Comment:
Staramy się nigdy nie używać divów w celach żeby coś wystylizować (tzn dodawać diva tylko po to żeby stylem nadać mu np. Border). Dlatego wszystko to załatwiamy zazwyczaj za pomocą :before i :after.
To samo dotyczy diva `experience__mini-box`. Zapamiętaj to tak że div to pudełko, które powinno zawierać w sobie coś.